scripts/main.js

```diff
@@ -71,4 +71,60 @@ const departments = {
     }
 }
 
-console.log(departments);
\ No newline at end of file
+console.log(departments);
+
+// opdracht 1
+console.log("De afdeling Sales heeft " + departments.sales.numberOfEmployees + " medewerkers");
+console.log("Marketing is een leuke afdeling om te werken. " + departments.marketing.description + ".");
+console.log("De afdeling Customer Service heeft " + departments["customer-service"].numberOfEmployees + " medewerkers.")
+console.log("Sales is een uitdagende afdeling om te werken als Verkoopmanager. " + departments.sales.jobs[1].description);
+
+const userInput = prompt("Over welke afdeling wil je meer informatie? Kies uit: [marketing / sales / customer-service]");
+console.log(userInput.toLowerCase());
+
+switch (userInput.toLowerCase()) {
+    case "marketing":
+        console.log("Je koos " + userInput.toLowerCase() + ". " + departments[userInput].description);
+        break;
+    case "sales":
+        console.log("Je koos " + userInput.toLowerCase() + ". " + departments[userInput].description);
+        break;
+    case "customer-service":
+        console.log("Je koos " + userInput.toLowerCase() + ". " + departments[userInput].description);
+        break;
```

Comment:
Dit zou je nog iets kunnen versimpelen waardoor code dubbeling voorkomen wordt
```suggestion
    case "marketing":
    case "sales":
    case "customer-service":
        console.log("Je koos " + userInput.toLowerCase() + ". " + departments[userInput].description);
        break;
```